temporalio/workflow.py

```diff
@@ -361,6 +361,14 @@ def _logger_details(self) -> Mapping[str, Any]:
             "workflow_type": self.workflow_type,
         }
 
+    def get_current_build_id(self) -> str:
+        """The Build ID of the worker which executed the current Workflow Task. May be undefined if the
```

Comment:
```suggestion
        """The Build ID of the worker which executed the current Workflow Task.

           May be undefined if the
```

Python likes to separate the summary from the details in my experience. (pedantic)